projects/aca-content/assets/i18n/en.json

```diff
@@ -209,6 +209,21 @@
               "ADVANCED_FILTERS": "Advanced Filters:",
               "RESET": "Reset",
               "RESET_ACTION": "Reset search filters",
+              "SAVE_SEARCH": {
+                "ACTION_BUTTON": "Save Search",
+                "MODAL_HEADER": "Save this search",
+                "NAME_LABEL": "Name",
+                "NAME_REQUIRED_ERROR": "This field is required",
+                "DESCRIPTION_LABEL": "Description",
+                "CANCEL_BUTTON": "Cancel",
```

Comment:
It looks like common translation, is that possible to reuse some existing key for that label (maybe there is some in ADF)?